Packs/GCP-IAM/Integrations/GCP-IAM/GCP-IAM.py

```diff
@@ -0,0 +1,1939 @@
+
+        return response
+
+    def gcp_iam_organization_role_permission_add_request(self, role_name, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_organization_role_permission_remove_request(self, role_name, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_organization_role_list_request(self, organization_name, included_permissions, limit, page):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_organization_role_get_request(self, role_name):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_organization_role_delete_request(self, role_name):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_create_request(self, project_name, role_id, description, title, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_update_request(self, role_name, description, title, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_permission_add_request(self, role_name, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_permission_remove_request(self, role_name, permission):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_list_request(self, project_name, included_permissions, limit, page):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_get_request(self, role_name):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_project_role_delete_request(self, role_name):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_testable_permission_list_request(self, resource_name, limit, page):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+    def gcp_iam_service_account_delete_request(self, service_account_name):
+        # TODO
+
+        response = self._http_request('GET')
+
+        return response
+
+
+def get_pagination_readable_message(header: str, limit: int, page: int) -> str:
+    """
+    Generate pagination commands readable message.
+    Args:
+        header (str): Message header
+        limit (int): Number of elements to retrieve.
+        page (int): Page number.
+
+    Returns:
+        str: Readable message.
+
+    """
+    readable_message = f'{header}\n Current page size: {limit}\n Showing page {page} out others that may exist.'
```

Comment:
This is the formulation we got from XSOAR for commands that use pagination.
Since the API does not give an indication of the total number of existing pages, XSOAR has requested to specify it in this way.

Reply:
wasn't aware of this, @DanilVil can you approve?

Reply:
This comment should be displayed if pagination arguments were used, unless the amount of results returned was lower than the entered limit - what suggest that are no additional pages out there...